cmd/download_headers.cpp

```diff
@@ -50,7 +50,10 @@ int main(int argc, char* argv[]) {
 
     CLI11_PARSE(app, argc, argv);
 
-    log::set_verbosity(log::Level::kTrace);
+    log::Settings settings;
+    settings.log_threads = true;
+    log::init(settings);
+    log::set_verbosity(log::Level::kInfo);
```

Comment:
This line is not needed. log::Settings ctor already defaults the verbosity to kInfo

Reply:
done